NRLMMD-GEOIPS/geoips
# Switch from GEOIPS_BASEDIR to GEOIPS_TESTDATA_DIR

# Related Issues
fixes NRLMMD-GEOIPS/geoips#57

# Testing Instructions
- Spot check scripts in `./geoips/tests/scripts`
- More thoroughly check the following:
    - `./setup/bash_setup/gitall`
    - `./docs/available_functionality.rst`
    - `./geoips/tests/utils/copy_diffs_for_eval.sh`
    - `./geoips/tests/utils/delete_diff_dirs.sh`
    - `./geoips/tests/utils/delete_files_from_repo.sh`

# Summary
### Update all uses of GEOIPS_BASEDIR/test_data to use GEOIPS_TESTDATA_DIR instead                                                                                                                                                                             
* Found all uses of GEOIPS_BASEDIR/test_data and replaced with GEOIPS_TESTDATA_DIR
  * Except where GEOIPS_BASEDIR/test_data is used to initialize GEOIPS_TESTDATA_DIR
